### DISCUSSION TITLE
Office_S_Tips/system/2019/10/04/windows_10-uaktualnienie_1903

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#25] [Komentarze]
# Office_S_Tips/system/2019/10/04/windows_10-uaktualnienie_1903

Kilka ciekawych ulepszeń w kolejnych wydaniach Win 10, dotyczące także pracy biurowej, np. wstawianie symboli, powiększanie czcionki ekranowej. * Szybka zmiana formatu daty z pomocą Powershell * Microsoft PowerToys

https://andrzejq.github.io/Office_S_Tips/system/2019/10/04/windows_10-uaktualnienie_1903.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
